20180224_kubernetes/index.html

```diff
@@ -0,0 +1,223 @@
+        </div>
+    </div>
+
+    <div id="history_7" class="step slide text_align" data-x="0" data-y="-2300">
+        <p>And we'll dive into today's most popular orchestration engine:</p>
+        <img src="static/history_7.png" />
+        <div class="notes">
+            <ul>
+                <li>Created by Google, heavily influenced by Google’s Borg system, announced in mid-2014</li>
+                <li>Kubernetes v1 released on July 21, 2015. Google partnered with the Linux Foundation to form the Cloud Native Computing Foundation (CNCF)</li>
+                <li>Now used by Rancher Labs, Red Hat, CoreOS, IBM and Oracle.</li>
+                <li>Became the de-facto container deployment system</li>
+            </ul>
+        </div>
+    </div>
+
+
+    <div id="title" class="step" data-x="0" data-y="0" data-scale="4">
+        <span class="try">What exactly is</span>
+        <h1>Kubernetes</h1>
+        <img src="static/title_interrogative.png" />
+    </div>
+
+    <div id="pods" class="step" data-x="850" data-y="3500" data-rotate="90" data-scale="5">
+        <p>
+        It has sets of <strong>containers</strong> like Docker called <strong>Pods</strong>
+        </p>
+        <img src="static/pods.svg" />
+        <div class="notes">
+            Group of containers that will be co-located on the same host machine and share the same resources and IP address.<br />
+            A pod can define a volume (local disk directory or network disk)<br />
+        </div>
+    </div>
+
+    <div id="nodes" class="step" data-x="3700" data-y="2800" data-rotate="180" data-scale="5">
+        <p>Pods run on <b>Nodes</b> <span class="thoughts">(worker machine)</span></p>
+        <img src="static/nodes.svg" />
+        <div class="notes">
+            A node is a worker machine in Kubernetes and may be a VM or physical machine, depending on the cluster.<br />
+            You might have heard of master/minions before, and nodes were previously called minions.
+            Multiple Pods can run on one Node.<br />
+            Every Kubernetes Node runs at least:
+            <ul>
+                <li>Kubelet, a process responsible for communication between the Kubernetes Master and the Node; it manages the Pods and the containers running on a machine.</li>
+                <li>A container runtime (like Docker, rkt) responsible for pulling the container image from a registry, unpacking the container, and running the application.</li>
+                <li>Containers should only be scheduled together in a single Pod if they are tightly coupled and need to share resources such as disk.</li>
+            </ul>
+        </div>
+    </div>
+
+    <div id="kubelet" class="step" data-x="3600" data-y="1600" data-z="-15000" data-rotate="300" data-scale="0.2">
+        <h1>Kubelet</h1>
+        <q>An agent that runs on each node in the cluster. It makes sure that containers* are running and healthy in a pod</q><br />
```

Comment:
I would restructure this: You already introduce this as communcation & manager. How about explaining what is going on, i.e.:

- "_Kubelet_ + _Pod Spec_ → Running Pod"
- "_Kubelet_ restarts broken pods"
- "_Kubelet_ reports to _Master_ 🤫" _(Note: "master was not introduced before this section")_